src/files/models/Dijkstra.java

```diff
@@ -37,18 +37,23 @@ public void startDijkstra() {
         //Vertex startNode = graph.getVertex("J");
         //Vertex endNode = graph.getVertex("F");
 
-        Pair<Integer, Map<Vertex, Vertex>> results = dijkstra(startNode, endNode);
+        Vertex result = dijkstra(startNode, endNode);
```

Comment:
Would be nice with something similar with the Pair. Like an arrayList of the Vertex's as we can get the distances or positions of x and y from the vertices in the arrayList.
(this is probably already done by you recently)

Reply:
wut?